ephios/api/views/events.py

```diff
@@ -96,3 +105,30 @@ class EventViewSet(viewsets.ReadOnlyModelViewSet):
         .prefetch_related(Prefetch("shifts__participations"))
         .order_by("start_time")
     )
+
+
+class ParticipationSerializer(serializers.ModelSerializer):
+    event_title = serializers.CharField(source="shift.event.title")
+    state = ChoiceDisplayField(choices=AbstractParticipation.States.choices)
+    duration = serializers.SerializerMethodField()
+
+    class Meta:
+        model = LocalParticipation
+        fields = [
+            "id",
+            "shift",
+            "event_title",
+            "state",
+            "comment",
+            "start_time",
+            "end_time",
+            "duration",
+        ]
+
+    def build_unknown_field(self, field_name, model_class):
+        if field_name in {"start_time", "end_time"}:
+            return self.build_property_field(field_name, model_class)
+        return super().build_unknown_field(field_name, model_class)
+
+    def get_duration(self, obj):
+        return obj.end_time - obj.start_time
```

Comment:
This is returned as a string right now? Wouldn't we want a number type for seconds?

Reply:
sure